.pre-commit-config.yaml

```diff
@@ -16,7 +16,6 @@
   hooks:
   - id: check-case-conflict
   - id: check-executables-have-shebangs
-    exclude: ^(install/|_unit-test/_test_setup.sh)
```

Comment:
Those files are not ever run as executables, they are only ever sourced - given they did not have a shebang, running them as executable would not have worked anyway. Instead of ignoring the check, I fixed it by removing the executable permission (chmod -x).